src/utils/mergeOptions.ts

```diff
@@ -1,7 +1,6 @@
 import ensureArray from './ensureArray.js';
-import deprecateOptions from './deprecateOptions.js';
```

Comment:
How's js even work here?

Reply:
I think `rollup-plugin-node-resolve` is the reason.

Reply:
See also [this snarky comment](https://github.com/rollup/rollup/blob/master/rollup.config.js#L38). 😉